discover/portal/articles/130-forms/99-ddl/06-creating-display-templates.markdown

```diff
@@ -80,8 +81,93 @@ display templates:
 2.  Click the widget's *Add Display Template* button. This opens the same form 
     as above for creating a display template for the list's definition. 
 
-<!-- 
-Per Russ, include a more detailed example of creating a template, when possible. 
-See example in the analogous 7.0.x article, and figure out which code isn't 
-specific to the cross-article Lunar Resort example in those docs. 
--->
+## Display Templae Editor
+
+Helper variables are available in the template editor. These provide access to
+most of the data that you'll use in creating Display Templates. The variables
+under the heading Data List Variables let you inject specific information about
```

Comment:
It makes sense. I think any confusion will be cleared up quickly once they hover over the variables or click them to add the script, but it might make sense to display the list like this instead:
>**Data Definition ID:** `reserved_ddm_structure_id`
>
>**Data List Description:** `reserved_record_set_description`
>
>**Data List ID:** `reserved_record_set_id`
>
>**Data List Name:** `reserved_record_set_name`
>
>**Template ID:** `reserved_ddm_template_id`

What do you think?

Reply:
Lovely!